doc/subsystems/logging/logger.rst

```diff
@@ -207,15 +229,36 @@ In order to use instance level filtering following steps must be performed:
 Note that when logger is disabled logger instance and pointer to that instance
 are not created.
 
-- logger can be used in function
+In order to use instance logging API in the source file a compile-time log level
```

Comment:
change to:  

    In order to use the instance logging API in a source file, a compile-time log level ...

Reply:
done